watchdog/Dockerfile.packager

```diff
@@ -0,0 +1,6 @@
+FROM openfaas/watchdog:build as build
+FROM scratch
+
+ARG HARDWARE
```

Comment:
Minor nit - since Windows isn't "HARDWARE" - platform may work better. This doesn't prevent a merge though.